odoo/sql_db.py

```diff
@@ -806,6 +809,7 @@ def connection_info_for(db_or_uri):
     :param str db_or_uri: database name or postgres dsn
     :rtype: (str, dict)
     """
+    app_name = "pid%d" % os.getpid()
```

Comment:
```suggestion
    app_name = "odoo-%d" % os.getpid()
```

This information is intended for PG logs, so the 2 key pieces of information are: the app (`odoo`) and the PID. Using a hyphen (`-`) rather than an underscore makes it look more like different/structured values IMO, but maybe that's just me.